charts/pulsar-admin-console/values.yaml

```diff
@@ -0,0 +1,118 @@
+#
+#
+# Copyright DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+replicaCount: 1
+
+image:
+  repository: datastax/pulsar-admin-console
+  pullPolicy: IfNotPresent
+  tag: ""
```

Comment:
I think there is a way to default this to the `appVersion` in the chart.yaml. Here is an example: https://github.com/apache/pulsar-helm-chart/pull/200

Reply:
it's already done, see here: https://github.com/datastax/charts/pull/7/files#diff-1ee99e4a17227a771b15de53aad243c423181a8d318b2f9e48a96ec854c74a17R61

Reply:
Thanks, I missed that!